packages/subapp-web/lib/init.js

```diff
@@ -34,12 +30,32 @@ module.exports = function setup(setupContext) {
     basePath: ""
   };
 
+  let inlineRuntimeJS = "";
+  let rtJsCdnEntryPoint;
+  if (process.env.NODE_ENV === "production") {
+    rtJsCdnEntryPoint = Object.keys(cdnJsBundles).filter(ep =>
+      cdnJsBundles[ep].includes("runtime.bundle")
+    )[0];
+    const runtimeFullPath =
+      rtJsCdnEntryPoint &&
+      cdnJsBundles[rtJsCdnEntryPoint] &&
+      Path.join(Path.resolve("dist"), "js", Path.basename(cdnJsBundles[rtJsCdnEntryPoint]));
```

Comment:
join and resolve together is redundant: https://nodejs.org/api/path.html#path_path_resolve_paths